docs/usage/http-client.md

```diff
@@ -84,6 +84,7 @@ Type notations are formatted like so:
 Property | Description
 --- | ---
 `query` | `Object=null`. When provided, HTTP Client serialize it and appends the `queryString` to `Request.url` property.
+`loadSpec` | `Boolean=undefined`. This property will be present and set to `true` when the `Request` was constructed internally by `SwaggerClient` to fetch the OAS definition befined by `url` or when resolving remote JSON References.
```

Comment:
typo: befined -> defined

Reply:
Fixed in 2e2e47c9f68b45d8e4ec52dc86f7f1b8357fea89